app/models/course/discussion/topic.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 class Course::Discussion::Topic < ActiveRecord::Base
   actable
+  class_attribute :global_topic_models
+  self.global_topic_models ||= []
```

Comment:
This is not just class name. If you change the class, the autoloader won't be able to apply your changes if the class reference is stashed.